wewatch-client/src/components/noMatch/noMatch.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState  } from "react";
+import { Redirect } from "react-router-dom";
+
+import './noMatch.css';
+
+const NoMatch = () => {
```

Comment:
can we get a screenshot in the PR of what this page looks like? :octocat:

Reply:
done!